networking/eth2/src/testFixtures/java/tech/pegasys/artemis/networking/eth2/Eth2NetworkFactory.java

```diff
@@ -152,11 +156,16 @@ private NetworkConfig generateConfig() {
     }
 
     private void setDefaults() {
+      if (diskUpdateChannel == null) {
+        final EventChannels eventChannels =
+            EventChannels.createSyncChannels(TEST_EXCEPTION_HANDLER, new NoOpMetricsSystem());
+        diskUpdateChannel = eventChannels.getPublisher(DiskUpdateChannel.class);
+      }
```

Comment:
This seems like a complicated way of creating a mock that will always return `new SafeFuture()` that doesn't complete.  Since there's no subscriber, nothing will reply so the future never completes.

Reply:
Agreed. Thanks for catching. Changed.